package.json

```diff
@@ -18,9 +18,8 @@
   "license": "GPL-3.0-only",
   "dependencies": {
     "@cospired/i18n-iso-languages": "2.0.5",
-    "@material-ui/core": "^3.9.3",
-    "@material-ui/icons": "^3.0.2",
-    "@material-ui/lab": "^3.0.0-alpha.30",
```

Comment:
@martinbedouret you removed `material-ui/lab`, are you sure we're not using any of the components?
![image](https://user-images.githubusercontent.com/6969966/72358684-a026b480-36f5-11ea-8a00-f4c3a2c30e95.png)